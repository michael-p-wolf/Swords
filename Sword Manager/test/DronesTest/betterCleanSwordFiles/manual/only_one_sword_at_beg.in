6 5
0, 10, 1, 1, 1, 2, 3, "3", "3", "3", "6"
-1, 10, 1, 1, 1, 2, 3, "3", "3", "3", "4"
-1, 10, 1, 1, 1, 2, 3, "3", "3", "3", "5"
-1, 10, 1, 1, 1, 2, 3, "3", "3", "3", "9"
-1, 10, 1, 1, 1, 2, 3, "3", "3", "3", "1"
-1, 10, 1, 1, 1, 2, 3, "3", "3", "3", "7"
0, 1, 2, 3, "7"
0, 1, 2, 3, "4"
0, 1, 2, 3, "5"
0, 1, 2, 3, "9"
0, 1, 2, 3, "1"
