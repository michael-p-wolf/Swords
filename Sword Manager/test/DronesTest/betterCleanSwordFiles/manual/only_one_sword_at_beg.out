0, 0, 1, 2, 3, "6"
10, 10, 1, 2, 3, "7"
10, 10, 1, 2, 3, "4"
10, 10, 1, 2, 3, "5"
10, 10, 1, 2, 3, "9"
